resources/eventing/charts/controller/templates/webhook.yaml

```diff
@@ -0,0 +1,56 @@
+{{ if .Values.enableNewCRDVersion }}
+---
+apiVersion: admissionregistration.k8s.io/v1
+kind: MutatingWebhookConfiguration
+metadata:
+  creationTimestamp: null
+  name: subscription-mutating-webhook-configuration
```

Comment:
(nit) is it not better to make it configurable as it is used in the code too

Reply:
done